Sources/NIO/EventLoop.swift

```diff
@@ -21,7 +21,6 @@ import Dispatch
 /// will be notified once the execution is complete.
 public struct Scheduled<T> {
     private let promise: EventLoopPromise<T>
-    private let cancellationTask: () -> Void
```

Comment:
Hopefully you filed a Swift bug about why the compiler didn't catch this for you :)